src/Pages/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "../components/ArticleCard";
+import { Container, Row, Col } from "react-bootstrap";
+
+const Home = () => {
+  const [articles, setArticles] = useState(null);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+
+  if (articles === null) {
+    return <h2>Fetching some articles...</h2>;
+  }
+
+  return (
+    <Container>
+      <h2 className="my-4">News Articles</h2>
+      <Row xs={1} sm={2} className="row-style">
+        {articles.map((article, index) => {
+          return (
+            <Col key={index}>
+              <ArticleCard article={article} />
+            </Col>
+          );
+        })}
+      </Row>
+    </Container>
+  );
+};
+
+export default Home;
```

Comment:
I love that you've used a 3rd party library to style this especially to make it dynamic, of course, you could change this later but consider all screen sizes if you are going to do that, there is some overlapping text. And consider that the picture sizes maybe different too which can throw off some of the grid sizes.


<img width="521" alt="Screenshot 2024-12-17 at 09 05 09" src="https://github.com/user-attachments/assets/70f4c7da-35d3-433e-8a38-6b3bb137145a" />